docs/source/starter/installation.rst

```diff
@@ -97,15 +97,30 @@ but this command will ensure that for everyone.
 **Note:** You will need to run ``conda activate geoips`` every time you want to
 run or work on GeoIPS.
 
-4. Clone the GeoIPS git repository, for installation and testing commands
+++++++++++++++++++++
```

Comment:
Do we want to have the pypi installation and the source installation in the same instructions or do we want to have separate set of instructions for non-developer pip installation?